web/src/beta/features/AssetsManager/AssetsSelector.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC, useCallback, useState } from "react";
+
+import { Button, Modal, ModalPanel } from "@reearth/beta/lib/reearth-ui";
+import { useT } from "@reearth/services/i18n";
+import { styled } from "@reearth/services/theme";
+
+import { Asset } from "./types";
+
+import AssetsManager, { AssetsManagerProps } from ".";
+
+export type AssetsSelectorProps = AssetsManagerProps & {
+  opened?: boolean;
+  onClose?: () => void;
+  onAssetSelect?: (url?: string, name?: string) => void;
+};
+
+const AssetsSelector: FC<AssetsSelectorProps> = ({ opened, onClose, onAssetSelect, ...props }) => {
+  const t = useT();
+
+  const [selectedAssets, setSelectedAssets] = useState<Asset[]>([]);
+
+  const handleSelectFinish = useCallback(() => {
+    if (!selectedAssets || selectedAssets.length === 0) {
+      onAssetSelect?.(undefined, undefined);
+    } else {
+      onAssetSelect?.(selectedAssets[0].url, selectedAssets[0].name);
+    }
+    onClose?.();
+  }, [selectedAssets, onAssetSelect, onClose]);
+
+  return (
+    <Modal size="large" visible={!!opened}>
+      <ModalPanel
+        title={t("Select Asset")}
```

Comment:
I think `Asset` should be lower-case but not important.

Reply:
Emmm we have a lot of cases writing like that. Maybe we can go though them together when finalize the i18n translations.

Reply:
Sounds good.